### DISCUSSION TITLE
Tree shaking after version 1.6.5

### DISCUSSION THREAD

**Discussion by djkunkel** [vueform/vueform#203] [Questions]
I'm upgrading my project to the latest VueForm, but any release after 1.6.5 (1.6.6+) seems to double the size of my bundle. I had followed the instructions here: https://vueform.com/docs/tree-shaking#how-to-tree-shake

Does it have something to do with how the package is now built with prepack? I see this in the diff between 1.6.5 and 1.6.6.
`"release": "npm run prepack && npm publish --access public"`

I will do more investigation, but I thought I'd go ahead and ask.

**Comment by nclemeur** [comment]
I am also interested to find out. I did try to implement tree shaking, but could not really notice any difference in the bundled size, but I did not investigate much further, assuming I had an issue in my setup.

**Comment by adamberecz** [comment]
It seems like the docs was indeed not correct after `1.6.6` and also the package needed a bit of tweaking. I've updated them, please upgrade to `1.9.4` and revisit:
https://vueform.com/docs/tree-shaking

Main changes:
- you need a couple of [dependencies](https://vueform.com/docs/tree-shaking#install-dependencies) installed
- needs [Vite plugin](https://vueform.com/docs/tree-shaking#add-vite-plugin)
- imports need to happen from `@vueform/vueform/src/core.js` and `@vueform/vueform/themes/*`

Let me know if you face any tree-shaking issues after this.
